example/standalone/main.tf

```diff
@@ -27,19 +25,18 @@ module "nifi" {
 }
 
 module "nifi_registry" {
-  source = "github.com/hannemariavister/terraform-nomad-nifiregistry?ref=0.1.0"
+  source = "github.com/hannemariavister/terraform-nomad-nifiregistry?ref=0.2.0"
 
   # nomad
   nomad_datacenters = ["dc1"]
   nomad_namespace   = "default"
-  nomad_host_volume = "persistence"
```

Comment:
Has to be deleted in main.tf and variables.tf as well
https://github.com/hannemariavister/terraform-nomad-nifi/blob/issue_38_refactor/main.tf
https://github.com/hannemariavister/terraform-nomad-nifi/blob/issue_38_refactor/variables.tf

Reply:
done !